sql/src/test/java/io/crate/integrationtests/TransportSQLActionTest.java

```diff
@@ -1440,12 +1441,11 @@ public void testGroupByOnIpType() throws Exception {
     }
 
     @Test
-    @UseJdbc(false) // geo_point support is missing
     public void testInsertAndSelectGeoType() throws Exception {
         execute("create table geo_point_table (id int primary key, p geo_point) with (number_of_replicas=0)");
         ensureYellow();
-        execute("insert into geo_point_table (id, p) values (?, ?)", new Object[]{1, new Double[]{47.22, 12.09}});
-        execute("insert into geo_point_table (id, p) values (?, ?)", new Object[]{2, new Double[]{57.22, 7.12}});
+        execute("insert into geo_point_table (id, p) values (?, ?)", new Object[]{1, new PGpoint(47.22, 12.09)});
```

Comment:
why is this change necessary and how does this work if it isn't executed with jdbc but with the transport client?